crowbar_framework/app/models/network_service.rb

```diff
@@ -413,8 +412,6 @@ def build_net_info(network, name, db = nil)
     db["network"].each { |k,v|
       net_info[k] = v unless v.nil?
     }
-    net_info["usage"]= network
-    net_info["node"] = name
```

Comment:
In https://github.com/crowbar/crowbar-core/pull/511/commits/02f6c3d782f795487518f116ceb8937860e41271, but that's part of the followup PR, indeed. I can move it here if you want, though.